test-app/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: '1.0'
+name: unittest-nodeapi
```

Comment:
Call this "node-client", as "unittest" is meaningless in how it's used in the Node/Java client projects.